### PR TITLE
Add WP Mail sender Handler

### PR DESCRIPTION
**Pull request by joehoyle**
This is just used for sending, not inbound.

### CODE REVIEW

File: library/Falcon/Handler/WPMail.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+				$message->get_html(),
+				$headers
+			);
+		}
+
+		return null;
+	}
+
+	public static function get_name() {
+		return 'WP Mail';
```
Comment (rmccue) [inline library/Falcon/Handler/WPMail.php]: This needs to be translatable.

File: library/Falcon/Handler/WPMail.php [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+					$headers['In-Reply-To'] = $original;
+				}
+			}
+
+			if ( ! empty( $options['references'] ) ) {
+				$references = implode( ' ', $options['references'] );
+				$headers['References'] = $references;
+			}
+
+			wp_mail(
```
Comment (rmccue) [inline library/Falcon/Handler/WPMail.php]: We should probably check the return value.

File: library/Falcon/Handler/WPMail.php [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+		$from = Falcon::get_from_address();
+		$author = $message->get_author();
+
+		$messages = array();
+		foreach ( $users as $user ) {
+
+			$headers = array(
+				'From'         => sprintf( '%s <%s>', $author, $from ),
+				'Reply-To'     => sprintf( '%s', $message->get_reply_address( $user ) ),
+				'Content-Type' => 'text/html; charset=UTF-8',
```
Comment (rmccue) [inline library/Falcon/Handler/WPMail.php]: This should probably use the charset option instead.

File: library/Falcon/Handler/WPMail.php [hunk 4]
```diff
@@ -0,0 +1,89 @@
+<?php
+
+/**
+ * WP Mail Handler
+ *
+ * @author Joe Hoyle <joe@huamanmade.co.uk>
```
Comment (rmccue) [inline library/Falcon/Handler/WPMail.php]: "huamanmade"

**Comment by rmccue** [comment]
This appears to be sending HTML as text.

**Comment by rmccue** [comment]
@joehoyle Did we want to use this, or not bother?

**Comment by rmccue** [comment]
@joehoyle can you take a look at this and refresh as necessary?

File: library/Falcon/Handler/WPMail.php (hunk 2 above)
Comment (joehoyle) [reply library/Falcon/Handler/WPMail.php]: Fixed

File: library/Falcon/Handler/WPMail.php (hunk 3 above)
Comment (joehoyle) [reply library/Falcon/Handler/WPMail.php]: As it happens, wp_mail will set the chartset, so I just removed the directive

**Comment by joehoyle** [comment]
> This appears to be sending HTML as text.

I verified the header is being set to `Content-Type: text/html; charset=UTF-8`, maybe I inadvertently fixed it.

File: library/Falcon/Handler/WPMail.php (hunk 4 above)
Comment (joehoyle) [reply library/Falcon/Handler/WPMail.php]: Fixed

File: library/Falcon/Handler/WPMail.php (hunk 1 above)
Comment (joehoyle) [reply library/Falcon/Handler/WPMail.php]: Fixed

**Pull request merged**